src/Command/InfectionCommand.php

```diff
@@ -115,7 +116,7 @@ protected function configure(): void
                 'test-framework',
                 null,
                 InputOption::VALUE_REQUIRED,
-                'Name of the Test framework to use (phpunit, phpspec)',
+                'Name of the Test framework to use (' . implode(', ', TestFrameworkTypes::TYPES) . ')',
```

Comment:
I think this could be done separately as well

Reply:
https://github.com/infection/infection/pull/839